src/module_object.cpp

```diff
@@ -362,6 +375,27 @@ Value ModuleObject::cvar_set(Env *env, SymbolObject *name, Value val) {
     return val;
 }
 
+Value ModuleObject::class_variable_get(Env *env, Value name) {
+    auto *name_sym = name->to_symbol(env, Conversion::Strict);
+
+    auto val = cvar_get_or_null(env, name_sym);
+    if (!val) {
+        env->raise_name_error(name_sym, "uninitialized class variable {} in {}", name_sym->string(), inspect_str());
+    }
+    return val;
+}
+
+Value ModuleObject::class_variable_set(Env *env, Value name, Value value) {
+    assert_not_frozen(env);
+    if (name->is_string()) {
+        name = name->as_string()->to_symbol(env);
+    } else if (!name->is_symbol()) {
+        name = name->to_str(env)->to_symbol(env);
+    }
```

Comment:
Should we use `Object::to_symbol()` here? I only ask because @herwinw has done some recent work to DRY up all these conversion bits of code and I *think* `to_symbol()` makes sense for this. (Not 100% sure though.)

Reply:
I think that's actually more correct. For example calling `class_variable_set` with `nil` as the first argument raises a wrong error in natalie currently. Changing it to use `Object::to_symbol()` fixes this!